caffe2/python/pybind_state.h

```diff
@@ -161,9 +174,10 @@ class TensorFeeder : public BlobFeederBase {
  public:
   void FeedTensor(
       const DeviceOption& option,
-      PyArrayObject* original_array,
+      PyObject* original_array,
```

Comment:
Hmm can we keep it as a `PyArrayObject*` but forward declare `struct PyArrayObject;` if `USE_NUMPY` is false? It's really much safer.

Reply:
Sure, I will do that. I had thoughts about this earlier, but then I looked at `torch/csrc/utils/tensor_numpy.*` and followed the convention there.

Reply:
@apaszke this is fixed.